0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -1,62 +1,127 @@
 # Simple Payment Setup Protocol (SPSP)
 
-The Simple Payment Setup Protocol (SPSP) is a basic application-layer Interledger protocol for wallet-to-wallet or wallet-to-merchant payments.
+## Preface
 
-## Stages
+This document describes the Simple Payment Setup Protocol (SPSP), a basic application-layer Interledger protocol for exchanging payment information between senders and recipients.
 
```

Comment:
Suggest:

This document describes the Simple Payment Setup Protocol (SPSP), a basic protocol for exchanging payment information between senders and recipients to setup an Interledger payment.